src/manufacturer/deleteManufacturerDialog.component.tsx

```diff
@@ -78,6 +79,7 @@ const DeleteManufacturerDialog = (props: DeleteManufacturerProps) => {
         <Button
           onClick={handleDeleteManufacturer}
           disabled={isDeletePending || error}
+          endIcon={isDeletePending ? <CircularProgress size={18} /> : null}
```

Comment:
```suggestion
          endIcon={isDeletePending ? <CircularProgress size={20} /> : null}
```
Why is this a different size to the other delete dialogues?

Reply:
Was not meant to be - probably was messing around trying to find the perfect size and forgot to change it back